src/style.css

```diff
@@ -60,7 +60,7 @@ a {
   overflow-y: scroll;
 }
 
-.table-row:not(:hover, :focus-within) .hover-only, .table-cell:not(:hover, :focus-within) .cell-hover-only {
+.table-row:not(:hover) .hover-only, .table-cell:not(:hover) .cell-hover-only {
```

Comment:
this is the primary fix: the icons only depend on whether the mouse is hovering over them; focus/blur does not matter.